.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: go
+
+go:
+- 1.9
+- tip
+
+before_install:
+- go get github.com/Masterminds/glide
+- go get github.com/alecthomas/gometalinter
+- go get -u github.com/go-swagger/go-swagger/cmd/swagger
+
+install:
+- glide install --strip-vendor
+- gometalinter --install
```

Comment:
I don't think `gometalinter --install` is needed as it's not required by `make test`.

Reply:
as above